src/client/rooms/room-page.js

```diff
@@ -83,7 +90,7 @@ function leaveRoom() {
 
 const mapStateToProps = (state, ownProps) => ({
     roomUsers: state.room.users,
-    roomData: state.room.data,
+    roomData: state.room.room,
```

Comment:
We'll sort this out when we do our front-end refactoring next week if that's okay?